.travis.yml

```diff
@@ -8,3 +8,13 @@ rvm:
 
 cache: bundler
 script: bundle exec middleman build
+deploy:
+  provider: s3
+  access_key_id:$AWS_ACCESS_KEY
```

Comment:
@mikestonecodes -- There needs to be a space here I guess before `$AWS_ACCESS_KEY`?